front/app/containers/LandingPage/SignedOutHeader/index.tsx

```diff
@@ -0,0 +1,29 @@
+import useAppConfiguration from 'hooks/useAppConfiguration';
+import React from 'react';
+import Layout1 from './Layout1';
+import { isNilOrError } from 'utils/helperUtils';
+import Outlet from 'components/Outlet';
+
+interface Props {}
+
+const SignedOutHeaderIndex = ({}: Props) => {
+  const appConfiguration = useAppConfiguration();
+
+  if (!isNilOrError(appConfiguration)) {
+    const homepageBannerLayout =
+      appConfiguration.data.attributes.settings.customizable_homepage_banner
+        ?.layout || 'layout_1';
```

Comment:
Hard-coded fallback when module is not enabled.

Reply:
What if `layout` was set, but then `customizable_homepage_banner` was disabled for this tenant?
Maybe we should better use 
```js
if appConfiguration.data.attributes.settings.customizable_homepage_banner.enabled
 ...
```

Reply:
> What if layout was set, but then customizable_homepage_banner was disabled for this tenant?
Maybe we should better use

Good remark. We'd have to check in the back-end if customizable_homepage_banner gets sent to the user if that feature is disabled. If it is sent, we should indeed add a feature flag (includes both allowed and enabled).

Reply:
I've just checked opening this url http://localhost:3000/web_api/v1/app_configuration
It's sent indeed
```json
          "layout": "layout_1",
          "allowed": false,
          "enabled": false,
```

Reply:
Okay, good catch 👍